engine/consensus/message_hub/message_hub.go

```diff
@@ -302,10 +293,14 @@ func (h *MessageHub) processQueuedVote(packed *packedVote) error {
 	return nil
 }
 
-// processQueuedProposal performs actual processing of flow.Header, as a result of successful invocation
-// broadcasts block proposal to consensus committee and rest of Flow network.
+// sendOwnProposal propagates the block proposal to the consensus cluster and submits to non-consensus network:
```

Comment:
### ❗ 
In this file, we repeatedly use the terms "cluster consensus" and "consensus cluster", which I find creates unnecessary ambiguity with the collector clusters. 

For the main consensus' `MessageHub` I would suggest to avoid the term "cluster" entirely:

```suggestion
// sendOwnProposal propagates the block proposal to the consensus committee and submits to non-consensus network:
```

For the other two occurrences of "cluster consensus" ([here](https://github.com/onflow/flow-go/blob/5dfb05ce21739c27046e14a497a61e5c99b02fb7/engine/consensus/message_hub/message_hub.go#L48) and [here](https://github.com/onflow/flow-go/blob/5dfb05ce21739c27046e14a497a61e5c99b02fb7/engine/consensus/message_hub/message_hub.go#L62)), we could just remove the word 'cluster'